EIPS/eip-4844.md

```diff
@@ -234,6 +232,44 @@ def get_origin(tx: SignedBlobTransaction) -> Address:
     return ecrecover(tx_hash(tx), int(sig.y_parity)+27, sig.r, sig.s)
 ```
 
+### Header extension
+
+The current header encoding is extended with a new 256-bit unsigned integer field `blobs`. This is the running
+total of excess blobs included on chain since this EIP was activated. If the total number of blobs is below the
+average, `blobs` is capped at zero.
+
+The resulting RLP encoding of the header is therefore:
+
+```
+rlp([
+    parent_hash,
+    ommers_hash,
+    coinbase,
+    state_root,
+    tx_root,
+    receipt_root,
+    bloom,
+    difficulty,
+    number,
+    gas_limit,
+    gas_used,
+    time,
+    extra,
+    mix_digest,
+    nonce,
+    base_fee,
+    blobs
+])
+```
+
+The value of `blobs` can be calculated using the parent header and number of blobs in the block.
+
+```python
+def calc_excess_blobs(parent: Header, blobs: int) -> int:
+    adjusted = parent.blobs + blobs
+    return adjusted - min(TARGET_BLOBS_PER_BLOCK, adjusted)
```

Comment:
suggested more expressive naming:
* `new_blobs` instead of `blobs`
* `new_total` instead of `adjusted`

Reply:
I think `new_total` is actually incorrect though, because it is the `excess_blobs` + `new_blobs` which isn't always the total number of blobs.